File: ProyectoMysql/server/BusinessLayer/UnidadMedida.py
from DataLayer.UbigeoDB import *
from EntityLayer.GeneralEntity import *

class UnidadMedida:
    def BuscarItem(Nombre : str):
        try:
            return UbigeoDB.BuscarItem(Nombre)
        except Exception as e:
            print(e)

    def ObtenerItems(UbigeoId : int):
        try:
            return UbigeoDB.ObtenerItem(UbigeoId)
        except Exception as e:
            print(e)

File: ProyectoMysql/Api/LogisticStorage/LogisticStorage.BusinessLayer/UnidadMedida.cs
using LogisticStorage.DataLayer;
using LogisticStorage.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LogisticStorage.BusinessLayer
{
    public class UnidadMedida
    {

        public static List<UnidadMedidaEntity> ObtenerItems()
        {
            UnidadMedidaDB DB = new UnidadMedidaDB();
            return DB.ObtenerItems();
        }

        public static List<UnidadMedidaEntity> ObtenerItem(Int32 UnidadMedidaId)
        {
            UnidadMedidaDB DB = new UnidadMedidaDB();
            return DB.ObtenerItem(UnidadMedidaId);
        }
    }
}
